packages/ngx-web-component/package.json

```diff
@@ -1,30 +1,15 @@
 {
   "name": "@readalongs/ngx-web-component",
   "type": "module",
-  "version": "1.1.1",
+  "version": "1.1.2",
+  "private": false,
   "peerDependencies": {
-    "@angular/animations": "^15.2.4",
-    "@angular/common": "^15.2.4",
-    "@angular/compiler": "^15.2.4",
-    "@angular/core": "^15.2.4",
-    "@angular/forms": "^15.2.4",
-    "@angular/platform-browser": "^15.2.4",
-    "@angular/platform-browser-dynamic": "^15.2.4",
-    "@angular/router": "^15.2.4",
-    "rxjs": "~7.5.0",
-    "tslib": "^2.3.0",
-    "zone.js": "^0.11.8"
+    "@angular/core": ">=13",
```

Comment:
I assume you confirmed that it works with 13 and 14 as well?

Reply:
LOL, in fact, the currently released version doesn't actually work outside our monorepo (with angular 15):

```
./src/app/app.module.ts:3:0-72 - Error: Module not found: Error: Can't resolve '@readalongs/web-component/loader' in '/home/dhd/work/ReadAlongs/testng/test-ngx-web-component/src/app'

./node_modules/@readalongs/ngx-web-component/esm2020/lib/ngx-web-component.module.mjs:4:0-72 - Error: Module not found: Error: Can't resolve '@readalongs/web-component/loader' in '/home/dhd/work/ReadAlongs/testng/test-ngx-web-component/node_modules/@readalongs/ngx-web-component/esm2020/lib'

Error: src/app/app.module.ts:4:38 - error TS2307: Cannot find module '@readalongs/web-component/loader' or its corresponding type declarations.

4 import { defineCustomElements } from "@readalongs/web-component/loader";
```

Reply:
Okay, confirmed that the generated code in `ngx-web-component` is *very* specific to a single version of Angular.  So no it doesn't work with 13 or 14, which is sad really, because there's no good reason for it.